src/style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+ul,
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Each block should be in a separate files

Reply:
changed